tests/unit/keyspace.tcl

```diff
@@ -306,7 +306,7 @@ start_server {tags {"keyspace"}} {
     test {COPY basic usage for ziplist hash} {
         r del hash1 newhash1
         r hset hash1 tmp 17179869184
-        assert_encoding ziplist hash1
+        assert_encoding listpack hash1
```

Comment:
if we run the entire test suite in two modes, we can have a utility proc named `assert_packed_encoding` which can verify it's the expected type (i.e. it won't take an encoding name argument)

Reply:
Yes, I will add these tests later, before not adding is to avoid the amount of code too much impact review.